### PR TITLE
Added logic to generate list of IP and Mac addresses

### PR DESCRIPTION
**Pull request by shubham-html-css-js**
We have a new test case where we want to analyse how expensive Yuptoo IP/MAC/NIC modifiers are.
In order to do that, we need to do some change in our tooling, where we are generating sufficient amount of IP and Mac addresses.
This PR is an attempt to do the same

### CODE REVIEW

**Comment by shubham-html-css-js** [comment]
Hi @jhutar . Can you check whether I am doing the right thing or not

**Review by jhutar — CHANGES_REQUESTED** [review]
Let's solve this on some other place - in generator. Some other code might be using these functions, so let's not change the functions here.

File: opl/generators/inventory_ingress.py [hunk 1]
```diff
@@ -79,9 +80,9 @@ def _data(self):
             'system_purpose': json.dumps(self._get_system_purpose()),
             'ansible': json.dumps(self._get_ansible()),
             'insights_id': self._get_uuid(),
-            'ipv4_addr': self._get_ipv4(),
-            'ipv6_addr': self._get_ipv6(),
-            'mac_addr': self._get_mac(),
+            'ipv4_addr': self._get_ipv4(self.ip_v4_addresses),
```
Comment (jhutar) [inline opl/generators/inventory_ingress.py]: Something like this:
```suggestion
            'ipv4_addr': [self._get_ipv4() for _ in range(self.ip_v4_addresses)],
```

File: opl/generators/inventory_ingress.py [hunk 2]
```diff
@@ -18,19 +18,20 @@ class InventoryIngressGenerator(opl.generators.generic.GenericGenerator):
 
     installed_packages = []
 
-    def __init__(self, count, fraction=1, relatives=100, addresses=3, packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
+    def __init__(self, count, fraction=1, relatives=100, ip_v4_addresses=10, ip_v6_addresses=10, mac_addresses=1, packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
```
Comment (jhutar) [inline opl/generators/inventory_ingress.py]: I would just keep current API we have as we might use it in some tests:
```
[jhutar@jhutar 2023-01-10T09:37:23 iperf]$ git grep 'inventory_ingress\..*('
host_synchronizer/script-populate.py:    payload_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
inventory/script-generate.py:    payload_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
xjoin/diff_sys_profile.py:    generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(count=count, template='inventory_ingress_puptoo_template.json.j2')
yupana/script-generate.py:            host_generator = opl.generators.inventory_ingress.PayloadRHSMGenerator(
```
To make it simple, I would just use addresses option to set number of all 3 addresses (IPv4, IPv6 and MAC).

File: opl/generators/inventory_ingress_RHSM_template.json.j2 [hunk 3]
```diff
@@ -5,9 +5,9 @@
         "org_id":"{{ orgid }}",
         "subscription_manager_id":"{{ subscription_manager_id }}",
         "bios_uuid":"{{ bios_uuid }}",
-        "ip_addresses":["{{ ipv4_addr }}","{{ ipv6_addr }}","127.0.0.1"],
+        "ip_addresses":{{ ipv4_addr + ipv6_addr }},
```
Comment (jhutar) [inline opl/generators/inventory_ingress_RHSM_template.json.j2]: Please make sure this produces valid JSON:
```
>>> mylist = ["1.1.1.1", "2.2.2.2"]
>>> print(mylist)
['1.1.1.1', '2.2.2.2']
>>> import json
>>> json.loads("['1.1.1.1', '2.2.2.2']")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib64/python3.11/json/__init__.py", line 346, in loads
    return _default_decoder.decode(s)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.11/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 2 (char 1)
```

File: opl/generators/inventory_ingress.py [hunk 4]
```diff
@@ -18,19 +18,19 @@ class InventoryIngressGenerator(opl.generators.generic.GenericGenerator):
 
     installed_packages = []
 
-    def __init__(self, count, fraction=1, relatives=100, addresses=3, packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
+    def __init__(self, count, fraction=1, relatives=100, addresses=10, mac_addresses=1,packages=500, template='inventory_ingress_RHSM_template.json.j2'):   # noqa: E501
```
Comment (jhutar) [inline opl/generators/inventory_ingress.py]: Please keep the default for `addresses=` the same as before.

**Review by jhutar — APPROVED** [review]
Thank you!

**Comment by jhutar** [comment]
Please also change other inventory_ingress_* templates if they need to be changed because of this change.

**Pull request merged**